python/cudf/cudf/core/frame.py

```diff
@@ -344,8 +413,17 @@ def _concat(cls, objs, axis=0, ignore_index=False, sort=False):
         if isinstance(objs[0].columns, pd.MultiIndex):
             out.columns = objs[0].columns
         else:
+            if inner_join_axis_one and ignore_index:
+                names = [i for i in range(len(names))]
             out.columns = names
 
+            if (
+                inner_join_axis_one
+                and ignore_index
```

Comment:
```suggestion
                and ignore_index
```
Is this `ignore_index` check needed here?

Reply:
Yes, the column names only change if ignore_index is True. If it is False, they retain their old names

Reply:
Actually, it might not be needed. I'll double-check this quickly in case.

Reply:
removed this!